test/e2e/network/network_policy.go

```diff
@@ -35,10 +36,10 @@ import (
 	e2enode "k8s.io/kubernetes/test/e2e/framework/node"
 	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
 	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
+
```

Comment:
it seems this needs a comment to make golint happy

Reply:
Ah ok lemme check, making coffeee :)

Reply:
fixed